tty.go

```diff
@@ -175,19 +175,19 @@ func (t *tty) ClosePostStart() error {
 func (t *tty) Close() error {
```

Comment:
Note that this function _never_ returns an error; wondering if it should either not have the error return, or if one (or more) of the errors should actually be returned (or a multi-error used)

Reply:
Currently no callers of this care about the error, so I'd make it not return anything.

Reply:
Having `Close()` return an error is mostly so that you can map it to `io.Closer` as well as being "good Go practice". I think there is actually a bug here that we aren't returning `Close()` errors from any of the `Close`s we call in the function, but since very few projects check `Close()` errors I guess it isn't that bad...

Reply:
Having `Close()` return an error is mostly so that you can map it to `io.Closer` as well as being "good Go practice". I think there is actually a bug here that we aren't returning `Close()` errors from any of the `Close`s we call in the function, but since very few projects check `Close()` errors I guess it isn't that bad... :man_shrugging: